Improve visibility of models gallery

Bultako opened the pull request:
This small PR intends to improve the visibility of models gallery adding a section **Buit-in models** in the Modelling and also in the Tutorials index pages of the docs.

adonath commented:
Thanks @Bultako! I assigned @luca-giunti as reviewer, as he proposed this change in the dev call...

adonath commented:
@Bultako I think one could even put an additional link at the front-page in the "Getting started" section e.g. between "Tutorials" and "How To".

Bultako commented:
> @Bultako I think one could even put an additional link at the front-page in the "Getting started" section e.g. between "Tutorials" and "How To".

Ah, you mean in the horizontal navigation menu.
Ok, done.

I have also renamed the Getting Started page into Start to not have the same name as the Getting Started group in the menu, and added a link in the Modeling and Fitting section of the Overview page.

luca-giunti commented on docs/getting-started.rst:
```diff
@@ -2,8 +2,8 @@
 
-Getting Started
-===============
+Start
```
Maybe "Documentation", or "Quickstart Guide"?

luca-giunti commented on docs/modeling/index.rst:
```diff
@@ -42,6 +42,13 @@ datasets and thus provide links, allowing e.g. to do a joint fit to multiple
+Built-in models
+===============
+
+Gammapy provides a series of spatial, spectral and temporal models.
```
Maybe series -> large choice ?

luca-giunti commented on docs/overview.rst:
```diff
@@ -130,7 +130,9 @@ Modeling and Fitting
 This includes spectral, spatial and temporal model classes to describe the gamma-ray sky.
-Gammapy also contains a complete API for model parameter handling and model fitting.
+Gammapy also contains a complete API for model parameter handling and model fitting,
+with a series of spatial, spectral and temporal models. You may check out the whole list
```
Same comment as above (series->large choice)

luca-giunti commented on docs/tutorials/index.rst:
```diff
@@ -101,6 +101,13 @@ Time
+Built-in models
+---------------
+
+Gammapy provides a series of spatial, spectral and temporal models.
```
Same comment as above (series->large choice)

luca-giunti reviewed (approved):
Thanks @Bultako! Very nice

luca-giunti commented:
Please @adonath merge it, if you also approve. Thanks!

adonath reviewed (approved):
Thanks @Bultako! No further comment from my side...
